### ISSUE TITLE
Why is 5G wifi traffic less likely to be detected compared to 2.4G?

### ISSUE THREAD

**Issue by jamesshao8** [the-tcpdump-group/libpcap#915]
Hi all,

I found that 5G wifi is less likely to be detected than 2.4g. We printed out stdout from pcap and examine the detected data packets. If our wifi device is in 2.4G it's updating very frequently, however if our wifi device is in 5G mode, the data isn't updating frequently, even if our wifi devices are transmitting high speed contents (making video call) . Is this because of the wifi standards? Or simply because we don't have a good wifi card?

Please give me some advice. Thanks.

I have tried with rtl8812 and ralink 3070. Also with intel built-in wifi.

Please give me some advice. Thanks.

**Comment by guyharris** [comment]
So presumably by "high speed" you mean "802.11ac or faster", because 802.11n supports the 2.4GHz band.

Are you certain that the video call is using 802.11ac?

**Comment by sbourland** [comment]
Is this simply a problem of channel hopping?  When running the capture, 
are you locking onto the correct channel?  Sorry if asking the obvious.

**Comment by jamesshao8** [comment]
@guyharris I think I am sniffing mainly 802.11g and 802.11n. At least I think they are the most common devices around me. But I didn't decode whether they are 802.11g n or ac. By high speed, I meant the packets show up from pcap.

@sbourland Yes, after a close look, I found it's a channel problem. To be more specific, I found that if I locked my wifi adapter onto a specific channel, the packets on that channel will all be captured with no problem. If I don't set up a specific channel, I can still capture some packets from all channels, however a lot of packets will be lost.
